src/MagnumPlugins/UfbxImporter/UfbxImporter.cpp

```diff
@@ -952,43 +1127,72 @@ Containers::String UfbxImporter::doMaterialName(UnsignedInt id) {
 Containers::Optional<MaterialData> UfbxImporter::doMaterial(UnsignedInt id) {
     ufbx_material *material = _state->scene->materials[id];
 
+    /* @todo Do this only in tests? At least only in debug/once */
+    validateMaterialMappings();
```

Comment:
Absolutely -- put it into a `UfbxMaterials.h` for example, wrap in `namespace Magnum { namespace Trade { namespace {` (including the anonymous one), include both here and in the test and have `validateMaterialMappings()` as a regular test case.